.github/workflows/cypress.tests.yml

```diff
@@ -0,0 +1,17 @@
+name: Cypress tests
+
+on: [push]
+
+jobs: 
+  cypress-run:
+   runs-on: ubuntu-latest
+   container: cypress/browsers:node12.18.3-chrome87-ff82
+   steps:
+     - name: Checkout
+       uses: actions/checkout@v2
+
+
+     - name: Cypress run
+       uses: cypress-io/github-action@v2
+       with:
+        browser: firefox
```

Comment:
What I meant by "cypress should be a separate job" is what you did with the `cypress-run` keyword here. In this case, you created a separate "workflow" with a job called `cypress-run` inside of it. What I was hoping for is one workflow called `test.action.yml` with two jobs inside that run in parallel: `build` and `cypress-run`.

You can read more about the definitions here: https://docs.github.com/en/actions/using-jobs/using-jobs-in-a-workflow